src/System.Management.Automation/engine/Interop/Windows/AssignProcessToJobObject.cs

```diff
@@ -9,7 +9,7 @@ internal static partial class Interop
 {
     internal static partial class Windows
     {
-        [LibraryImport("Kernel32.dll", SetLastError = true)]
+        [LibraryImport("api-ms-win-core-job-l2-1-0.dll", SetLastError = true)]
```

Comment:
Are you sure we should depend on the specific API set DLL? Documentation recommends to link with kernel32.dll.